doc/tutorials/using_grub.rst

```diff
@@ -191,3 +192,35 @@ Here we provide another simple method to build GRUB in efi application format:
 #. Copy ACRN binary and guest kernel images to the GRUB-configured folder, e.g. ``/boot/`` folder on ``/dev/sda3/``;
 
 #. Run ``/EFI/boot/grub_x86_64.efi`` in the EFI shell.
+
+Enable quirk-modules-after-kernel for Multiboot2
+************************************************
+
+With the released GRUB (up to v2.06), SOS may not boot if the size of the SOS
```

Comment:
change ``SOS`` to ``the Service VM may not boot if the size of the Service VM OS``